src/main/scala/cosas/typeSets.scala

```diff
@@ -230,10 +230,6 @@ object typeSets {
 
     def toMap[K, V](implicit toMap: ToMap[S, K, V]): Map[K, V] = toMap(s)
 
-    def parseFrom[X](x: X)(implicit parser: S ParseFrom X): parser.Out = parser(s, x)
-
-    def serializeTo[X](implicit serializer: S SerializeTo X): X = serializer(s)
-
```

Comment:
Why aren't you sure? If it's a set of denotations, it works, if there is at least one random thing in it, it won't work — it's a very simple check.